sympy/stats/tests/test_continuous_rv.py

```diff
@@ -262,11 +261,20 @@ def test_cauchy():
 
 
 def test_chi():
+    from sympy import I
     k = Symbol("k", integer=True)
 
     X = Chi('x', k)
     assert density(X)(x) == 2**(-k/2 + 1)*x**(k - 1)*exp(-x**2/2)/gamma(k/2)
 
+    # Tests the characteristic function
+    # Passes in python3 but fails in python2
+    # C = sqrt(2)*I*x*gamma(k/2 + 1/2)*hyper((k/2 + 1/2,), (3/2,), -x**2/2)/gamma(k/2) + hyper((k/2,), (1/2,), -x**2/2)
```

Comment:
Most likely this fails due to integer division. You need to use e.g. S(1)/2 instead of 1/2

Reply:
ok, I will try that